client-linuxapp/src/reencrypt/mod.rs

```diff
@@ -64,7 +64,7 @@ fn perform_reencrypt(dev_name: &str) -> Result<()> {
         .context("Error opening device")?;
 
     dev.context_handle()
-        .load::<()>(None, None)
+        .load::<libcryptsetup_rs::CryptParamsLuks2Ref>(None, None)
```

Comment:
```suggestion
        .load::<libcryptsetup_rs::CryptParamsLuks2>(None, None)
```

Note: no idea if that'll work, but just a feeling.

Reply:
not really, after the bump to 0.8.0 there were some API changes, and that requires specifically specifying which parameters do you want in order to load the header. I changed it to `()` which turned out that was wrong and yielded the bugzilla, the `CryptParamsLuks2Ref` option is what I've tested that works for us